godel_robots/abb/godel_irb2400/godel_irb2400_support/launch/irb2400_blending.launch

```diff
@@ -4,6 +4,8 @@
   <arg name="sim_robot" default="true"/>
   <arg name="robot_ip" unless="$(arg sim_robot)" />
   <arg name="sim_sensor" default="true"/>
+  <arg name="sim_laser" default="true"/>
+  <arg name="laser_ip" unless="$(arg sim_laser)" default="192.168.32.50"/>
```

Comment:
yikes, hard-coded IP address.  I guess this is an accepted practice.  I also see that we do this in the following locations:
https://github.com/ros-industrial-consortium/godel/blob/kinetic-devel/godel_surface_detection/launch/keyence.launch#L5